### PR TITLE
making Staging kafka and zookeeper multi-az

### PR DESCRIPTION
**Pull request by prem-fissionhq**
https://dimagi-dev.atlassian.net/browse/SAAS-12484

### CODE REVIEW

**Review by shyamkumarlchauhan — APPROVED** [review]

**Comment by prem-fissionhq** [comment]
@shyamkumarlchauhan Could you please review this PR and merge this if everything looks good? 
I will decommission Kafka3,4 servers.  
@dannyroberts

**Review by shyamkumarlchauhan — APPROVED** [review]

**Comment by sravfeyn** [comment]
These kind of PR changes (the other [example](https://github.com/dimagi/commcare-cloud/pull/5169/files#r911632281)) where the installation scripts are modified and tested only with staging config, they are breaking for the default config which is used by all other envs and new envs. The scripts shouldn't get broken for default config values. Can you please also update the default  default config values or make the scripts handle both the configs? 

@prem-fissionhq

**Comment by sravfeyn** [comment]
These kind of PR changes (the other [example](https://github.com/dimagi/commcare-cloud/pull/5169/files#r911632281)) where the installation scripts are modified and tested only with staging config, they are breaking for the default config which is used by all other envs and new envs. The scripts shouldn't get broken for default config values. Can you please also update the default  default config values or make the scripts handle both the configs? 

@prem-fissionhq

File: src/commcare_cloud/ansible/roles/zookeeper/templates/zookeeper-server.service.j2 [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+WorkingDirectory=/opt/zookeeper
+User=root
+Group=root
+LimitNOFILE=32000
+Restart=on-failure
+Environment="ZOO_LOG_DIR={{ zookeeper_log_dir }}"
+
+ExecStart=/opt/zookeeper/bin/zkServer.sh start /opt/zookeeper/conf/zoo.cfg
+ExecStop=/opt/zookeeper/bin/zkServer.sh stop /opt/zookeeper/conf/zoo.cfg
+ExecReload=/opt/zookeeper/bin/zkServer.sh restart /opt/zookeeper/conf/zoo.cfg
```
[the remark this answers is not in the document]
Comment (sravfeyn) [reply src/commcare_cloud/ansible/roles/zookeeper/templates/zookeeper-server.service.j2:18]: It looks like that file is getting created only for zookeeper cluster setups, due to this change https://github.com/dimagi/commcare-cloud/pull/5154/files#diff-dfa9cf526240d01a974d273dd82d5979b3ce54cd023dd385ab9bea96598dac27R21

@prem-fissionhq This file should be generated regardless of zookeeper_cluster, right?

[the remark this answers is not in the document]
Comment (dannyroberts) [reply src/commcare_cloud/ansible/roles/zookeeper/templates/zookeeper-server.service.j2:18]: @shyamkumarlchauhan FYI

[the remark this answers is not in the document]
Comment (sravfeyn) [reply src/commcare_cloud/ansible/roles/zookeeper/templates/zookeeper-server.service.j2:18]: Fix here https://github.com/dimagi/commcare-cloud/pull/5359#pullrequestreview-1030262752

**Comment by mkangia** [comment]
Hey @prem-fissionhq 

Running `cchq <env> service kafka status` is giving an error for zookeeper which looks like is coming because the service check is using zookeeper instead of zookeeper-server.
Can you help update that and anywhere else this needs to change now? For instance, another place would be in `src/commcare_cloud/ansible/ping.yml`

**Comment by skodde** [comment]
> Running `cchq <env> service kafka status` is giving an error for zookeeper which looks like is coming because the service check is using zookeeper instead of zookeeper-server. Can you help update that and anywhere else this needs to change now? For instance, another place would be in `src/commcare_cloud/ansible/ping.yml`

Addressed in #5675.

**Pull request merged**